react-app/package.json

```diff
@@ -33,8 +33,9 @@
     "react-error-boundary": "4.0.12",
     "react-hook-form": "7.52.0",
     "react-leaflet": "4.2.1",
-    "react-router-dom": "6.24.0",
+    "react-router-dom": "6.25.0",
     "supercluster": "8.0.1",
+    "ts-node": "10.9.2",
```

Comment:
I think it was only in the API that we needed to move this up.

Reply:
moved back down